include/Realm.h

```diff
@@ -438,6 +440,11 @@ class Realm {
   
   EquationSystems equationSystems_;
 
+  //single-DOF graph
+  Teuchos::RCP<CrsGraph> scalarGraph_;
+  //multiple-DOF graph
+  Teuchos::RCP<CrsGraph> systemGraph_;
+
```

Comment:
We have one instance of a linear system a "tensor" for the dof, so 4 components in 2D or 9 in 3D.